### ISSUE TITLE
Port AbyssBox's song looping options

### ISSUE THREAD

**Issue by bsoap-cat** [ultraabox/ultrabox_typescript#185]

**Comment by choptop84** [comment]
AbyssBox 1.3 isn't out yet, this will be added after that update is released

**Comment by bsoap-cat** [comment]
> AbyssBox 1.3 isn't out yet, this will be added after that update is released

what? i was talking about this
![image](https://github.com/ultraabox/ultrabox_typescript/assets/127647009/f4aa7f0d-fbc2-4a98-ac8b-92b9577ff0e1)

**Comment by choptop84** [comment]
Yeah this is in the testing version. If you were on the official version you would find its not there [https://choptop84.github.io/abyssbox-app/](https://choptop84.github.io/abyssbox-app/)

**Comment by choptop84** [comment]
Update on this nearly a year later, AbyssBox 1.6 changes the song looping options to make it so then it persists when you share your song. Effectively meaning that this now would probably require a song version bump or something. Won't add the tag myself just bringing it up.
